specification/imds/resource-manager/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,725 @@
+                    "description": "This is the product of the image from the Marketplace."
+                }
+            }
+        },
+        "publicKeysProperties": {
+            "type": "object",
+            "properties": {
+                "path": {
+                    "type": "string",
+                    "description": "This specifies the full path on the VM where the SSH public key is stored."
+                },
+                "keyData": {
+                    "type": "string",
+                    "description": "This is the SSH public key certificate used to authenticate with the VM."
+                }
+            }
+        },
+        "ipv4Properties": {
+            "type": "object",
+            "properties": { 
+                "privateIpAddress": {
+                    "type":"string",
+                    "description":"This is the private IP address assigned to the interface."
+                 },
+                "publicIpAddress": { 
+                    "type":"string",
+                    "description":"This is the public IP address assigned to the interface."
+                }
+            }
+        },
+        "ipv6Properties": {
+            "type": "object",
+            "properties": { 
+                "privateIpAddress": {
+                    "type":"string",
+                    "description":"This is the private IPV6 address assigned to the interface."
+                }
+            }
+        },
+        "subnetProperties": {
+            "type": "object",
+            "properties": { 
+                "address": {
+                    "type":"string",
+                    "description":"This is the address range of the subnet."
+                },
+                "prefix": { 
+                    "type":"string",
+                    "description":"This is the prefix of the subnet."
+                }
+            }
+        },
+        "instance": {
+            "type": "object",
+            "properties": {
+                "compute": {
+                    "$ref": "#/definitions/compute",
+                    "description": "Compute Metadata"
+                },
+                "network": {
+                    "$ref": "#/definitions/network",
+                    "description": "Network Metadata" 
+                }
+            }
+        },
+        "attested": {
+            "type": "object",
+            "properties": {
+                "signature": {
+                    "type": "string",
+                    "description": "This is the encoded string containing the VM ID, plan information, and nonce value."
+                },
+                "encoding": {
+                    "type": "string",
+                    "description": "This is the encoding scheme of the signature."               
+                }
+            }
+        },
+        "ErrorResponse": {
+            "type": "object",
+            "properties": {
+                "error": {
```

Comment:
The Error response does not follow the ARM guidelines for Error definition.
It must follow this shape as described [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content).

Reply:
@ravbhatnagar - Can you comment on this?

Reply:
This is not an ARM API. It should be moved to a /data-plane subdirectory in the repo. Once that is done, we can remove ARM sign-off requirements

Reply:
Do you mean simply moving the files to imds/data-plane from imds/resource-manager?

Reply:
Done.